azure-pipelines.yml

```diff
@@ -11,26 +11,72 @@ pr:
       - develop
   drafts: false
 
-pool:
-  vmImage: ubuntu-latest
+variables:
+  environmentName: 'team-up-dev'
```

Comment:
replace the value with a variable called envName and set its default value to 'dev' trough Azure Pipelines UI